addOns/graphql/src/main/zapHomeFiles/scripts/scripts/variant/GraphQL Support.js

```diff
@@ -96,11 +98,11 @@ function getLeafName(helper, nodeName, msg) {
 function getTreePath(helper, msg) {
 	var query = getQuery(msg);
 	if (query == null) {
-		print("  not GraphQL");
+		// print("  not GraphQL");
 		return null;	
 	}
 	var uri = msg.getRequestHeader().getURI();
-	var path = uri.getPath().split('/');
+	var path = uri.getPath() != null ? uri.getPath().split('/') : '';
```

Comment:
`''` → `[]` ?

Reply:
`path.length` will return zero in either case but `[]` does seem more appropriate.

Reply:
Right, was more to have the same type.